src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './NewTodo.css';
+
+class NewTodo extends React.PureComponent {
+  state = {
+    selectUserId: 0,
+    inputTitle: '',
+    isErrorInput: false,
+    isErrorSelect: false,
+  }
+
+  handleChangeUser = (evt) => {
```

Comment:
не сокращай названия
```suggestion
  handleUserSelect = (event) => {
```